### PR TITLE
[Feature]Display readme in the Bento API server index page

### PR DESCRIPTION
**Pull request by yubozhao**
## Description

#1193 

Add a READMe section for the API server to display BentoService's docstring. Users can write markdown compatible information to the BentoService docstring, see the example below

```python
class IrisClassifier(BentoService):
    """
## Iris classifier readme

You can use markdown syntax

List:

* list item
* list item

1. abc
2. 123

Text: *Italic*  **bold**

Link: [BentoML](http://github.com/bentoml/bentoml)
    """
```
Screen Shot 2020-12-16 at 4 38 35 PM

The new index page has tabs selector for Swagger UI (default) and Readme 

Screen Shot 2021-01-05 at 5 49 56 PM

Screen Shot 2021-01-05 at 5 50 01 PM

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] New feature and improvements (non-breaking change which adds/improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code Refactoring (internal change which is not user facing)
- [ ] Documentation
- [ ] Test, CI, or build

## Component(s) if applicable
- [ ] BentoService (service definition, dependency management, API input/output adapters)
- [ ] Model Artifact (model serialization, multi-framework support)
- [x] Model Server (mico-batching, dockerisation, logging, OpenAPI, instruments)
- [ ] YataiService gRPC server (model registry, cloud deployment automation)
- [ ] YataiService web server (nodejs HTTP server and web UI)
- [ ] Internal (BentoML's own configuration, logging, utility, exception handling)
- [ ] BentoML CLI

## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
     'contextvars;python_version < "3.7"',
     'dataclasses;python_version < "3.7"',
     "chardet",
+    "markdown",
```
Comment (parano) [inline setup.py:54]: This could be done on the frontend right?

Comment (yubozhao) [reply setup.py:54]: I think the browser needs to install a markdown extension to render it.  This PR sends rendered markdown as HTML down to the clients.

File: bentoml/server/api_server.py [hunk 2]
```diff
@@ -221,6 +227,7 @@ def setup_routes(self):
         /feedback       Submitting feedback
         /metrics        Prometheus metrics endpoint
         /metadata       BentoService Artifact Metadata
+        /readme         BentoService readme info
```
Comment (parano) [inline bentoml/server/api_server.py:230]: why is this a separate route.. just display it in the index page

File: setup.py (hunk 1 above)
Comment (parano) [reply setup.py:54]: You should be able to include a JS library for rendering markdown on frontend only. I'm fine either way.

**Pull request merged**